_data/collection-index.yml

```
@@ -608,3 +608,8 @@
   contact: https://github.com/joshspicer/more-features/issues
   repository: https://github.com/joshspicer/more-features
   ociReference: ghcr.io/joshspicer/more-features
+- name: Features by MarlonPassos-git
+  maintainer: MarlonPassos-git
+  contact: hhttps://github.com/MarlonPassos-git/dev-container-features/issues
+  repository: https://github.com/MarlonPassos-git/dev-container-features
+  ociReference: ghcr.io/MarlonPassos-git/dev-container-features
```

```suggestion
  ociReference: ghcr.io/marlonpassos-git/dev-container-features
```

Hi 👋

The ORAS spec requires the `ociReference` to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.

---

Of course, thanks for the feedback